src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -100,19 +103,18 @@ public void init() {
     }
 
     private void cleanDatabases() {
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, aggregatedCollectionName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, waitlistCollectionName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, subscriptionCollectionName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, eventObjectMapCollectionName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, subscriptionCollectionRepatFlagHandlerName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, failedNotificationCollectionName);
-        mongoDBHandler.dropCollection(EIFFEL_INTELLIGENCE_DATABASE_NAME, sessionsCollectionName);
+        mongoDBHandler.dropCollection(database, aggregatedCollectionName);
+        mongoDBHandler.dropCollection(database, waitlistCollectionName);
+        mongoDBHandler.dropCollection(database, subscriptionCollectionName);
+        mongoDBHandler.dropCollection(database, eventObjectMapCollectionName);
+        mongoDBHandler.dropCollection(database, subscriptionCollectionRepatFlagHandlerName);
+        mongoDBHandler.dropCollection(database, failedNotificationCollectionName);
+        mongoDBHandler.dropCollection(database, sessionsCollectionName);
     }
 
     /*
-     * setFirstEventWaitTime: variable to set the wait time after publishing the
-     * first event. So any thread looking for the events don't do it before actually
-     * populating events in the database
+     * setFirstEventWaitTime: variable to set the wait time after publishing the first event. So any
+     * thread looking for the events don't do it before actually populating events in the database
      */
     private int firstEventWaitTime = 0;
```

Comment:
I know this is legacy but could you please move this to the top together with the other variables.

Reply:
Done.